napari/_qt/qt_event_loop.py

```diff
@@ -191,6 +194,8 @@ def get_app(
         perf_config.patch_callables()
 
     if not _app_ref:  # running get_app for the first time
+        from napari import _app  # noqa   # TEMPORARY
```

Comment:
temporary because we won't always 'bootstrap the app' from here?

Reply:
ah, well spotted.  thank you. I think that was a note to myself as I just threw things in various places.

The deal here is that _something_ needs to import `napari._app` to create the `Application` instance (but one needs to be a little careful about circular imports).  Since at the moment it's pretty strictly related to using napari in a gui context, I just threw it in here, where we create the Qt app for the first time..  I think it might not actually be a bad place to put it for now, but ultimately I envision this new `app_model` thing getting the qt app (rather than the current vice versa situation).  

Anyway i wanted to keep the footprint minimal for now, so this is where I bootstrapped it

Reply:
Cool! Makes a lot of sense

Reply:
I prefer the getter function instead of the side effects of import.

Reply:
i agree.  lemme rework

Reply:
looks better now, thanks both